tests/integration_tests/batch_scripts/summit.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Begin LSF Directives
+#BSUB -P GEO111
+#BSUB -W 0:30
+#BSUB -nnodes 1
+#BSUB -alloc_flags gpumps
+#BSUB -J rp_integration_test
+#BSUB -o rp_integration_test.%J
+#BSUB -e rp_integration_test.%J
+
+TEST="radical.pilot/tests/integration_tests/test_rm/test_lsf.py
+      radical.pilot/tests/integration_tests/test_lm/test_jsrun.py"
+
+cd $MEMBERWORK/geo111/integration_tests/
+rm -rf radical.pilot testing *.log
+git clone --branch devel https://github.com/radical-cybertools/radical.pilot.git
+
+
+module unload xl
+module unload xalt
+module unload spectrum-mpi
+module unload py-pip
+module unload py-virtualenv
+module unload py-setuptools
+module load gcc/8.1.1
+module load zeromq/4.2.5
+module load python/3.7.0-anaconda3-5.3.0
+
+conda create -p testing python=3.7 pytest PyGithub -y -c conda-forge
+
+source activate $PWD/testing
```

Comment:
`conda activate $PWD/testing`

Reply:
`conda activate` produces this error:
```
bash-4.2$ conda activate $PWD/testing

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.
```

I did `conda init` but still I got the same error. `source` on the other hand correctly activates the VE. I am not sure if it makes sense to change my .`bashrc` to support `conda` since `source` works. Do you have another idea?

Reply:
try to add this line `eval "$(conda shell.posix hook)"` before the activation line

Reply:
I added it before the VE creation and it seems to work.